commercetools.Sdk/commercetools.Sdk.HttpApi.IntegrationTests/Product/ProductFixture.cs

```diff
@@ -115,11 +124,19 @@ public Product CreateProductAndPublishIt(bool withVariants = false)
             return product;
         }
 
+        public Product CreateProductAndPublishIt(ProductType productType, bool withVariants = false)
+        {
+            Category category = this.CreateNewCategory();
+            var product = this.CreateProduct(category, productType, withVariants);
+            product = this.Publish(product);
```

Comment:
the ProductDraft has a property `publish` to publish a product on creation directly. Maybe you could change to code to use this property.

Reply:
yes it's better, I just used it instead of Publish Update Action